spec.html

```diff
@@ -31173,7 +31173,7 @@ <h1>Assertion</h1>
         <emu-alg>
           1. Return an internal AssertionTester closure that takes a State argument _x_ and performs the following steps:
             1. Let _e_ be _x_'s _endIndex_.
-            1. If _e_ is zero, return *true*.
+            1. If _e_ is 0, return *true*.
```

Comment:
Why is this a plain 0?

Reply:
Since the merge of PR #1135, [5.2.5 Mathematical Operations](https://tc39.es/ecma262/#sec-mathematical-operations) says
> The Number value 0, alternatively written 0𝔽, is defined as the double-precision floating point positive zero value. In certain contexts, it may also be written as +0 for clarity.

So in the context of the above step, `0` and `*+0*` would mean the same thing.

Reply:
Okay, and in the other contexts there is ambiguity? E.g., there's one set of steps that has n > 0 and then a following step with n is +0. It seems a little arbitrary.

Reply:
In cases where this PR has used `*+0*`, I don't think `0` would have been ambiguous.

@ljharb, do you have a criterion/guideline for choosing between `*+0*` and `0`?

Reply:
This is likely an oversight; I've tried to avoid ever having an unsigned `0`. I'll change this to `+0`.